src/redux/reducers/todoReducer.ts

```diff
@@ -29,6 +29,14 @@ export const todoReducer = (
       return { ...state };
     }
 
+    case TodoActions.DELETE_TODO: {
+      const updatedList = state.todoList.filter(
+        (todo) => todo.id !== action.payload,
+      );
+
+      return { ...state, todoList: [...updatedList] };
```

Comment:
updatedList is already a new array, why do you need to use spread operator ?

Reply:
Changed spredded updatedList to just passing the link for updatedList.